### DISCUSSION TITLE
Addition of min-height classes for the new container queries plugin

### DISCUSSION THREAD

**Discussion by midblep** [tailwindlabs/tailwindcss#10076] [Ideas]
Min width is a common usecase for mobile responsiveness, but there are cases when using min-height on a @container might also be useful, for example when something is a h-screen and you want to make sure stuff can't overflow. Would this be a possibility to add to the first-party plugin for container queries?

**Comment by ProPhycient5** [comment]
Hi @midblep, have you tried with this classname `min-h-screen` from the top div (or container)? I think this will help to cover the whole mobile screen and won't overflow if the content is not more than the mobile screen.

**Comment by kieranm** [comment]
I've opened a PR here to add support for this:
https://github.com/tailwindlabs/tailwindcss-container-queries/pull/7

Will wait for feedback from the tailwind team

**Reply by midblep** [reply]
Amazing, thanks so much.

**Reply by JOSEJESUSOCHOA** [reply]
The PR was closed without a merge :(

**Reply by midblep** [reply]
That's a shame, it looked great
